spec/services/reports/reports_creator_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Reports::ReportsCreator do
+  let(:legal_aid_application) { create :legal_aid_application, :with_proceeding_types, :with_everything, state: :generating_reports }
+
+  subject { described_class.call(legal_aid_application) }
+
+  describe '.call' do
+    it 'attaches merits_report.pdf to the application' do
```

Comment:
As this tests for both reports, would `attaches merits_report.pdf and means_report.pdf to the application` be more accurate?

Reply:
yes definitely 👍